app/code/Magento/Catalog/ViewModel/Product/Breadcrumbs.php

```diff
@@ -39,20 +44,21 @@ class Breadcrumbs extends DataObject implements ArgumentInterface
     /**
      * @param Data $catalogData
      * @param ScopeConfigInterface $scopeConfig
-     * @param Json|null $json
+     * @param JsonHexTag $jsonSerializer
      * @param Escaper|null $escaper
      * @SuppressWarnings(PHPMD.UnusedFormalParameter)
      */
     public function __construct(
         Data $catalogData,
         ScopeConfigInterface $scopeConfig,
-        Json $json = null,
+        JsonHexTag $jsonSerializer,
```

Comment:
Original class was kept for backward compatibility reasons, please do not remove it.
Could you introduce new optional parameter for JsonHexTag object?

Reply:
@ihor-sviziev Thanks for the suggestion, As suggest I have updated.

Reply:
Please make new argument as optional, move to last position and get it from ObjectManager as fallback